libraries/AP_Proximity/AP_Proximity_Backend.cpp

```diff
@@ -31,6 +33,10 @@ AP_Proximity_Backend::AP_Proximity_Backend(AP_Proximity &_frontend, AP_Proximity
 {
     // initialise sector edge vector used for building the boundary fence
     init_boundary();
+
+    obstacle_distance.increment = 360/PROXIMITY_NUM_SECTORS;
```

Comment:
let's always set the increment_f as well just to be nice to the consumer.

Reply:
Done.